tests/test_form_import.py

```diff
@@ -0,0 +1,192 @@
+from dataclasses import asdict
+from unittest import mock
+from uuid import uuid4
+
+import pytest
+
+from app.db.models.application_config import Component
+from app.db.models.application_config import Lizt
+from app.import_config.load_form_json import _build_condition
+from app.import_config.load_form_json import _find_list_and_create_if_not_existing
+from app.import_config.load_form_json import add_conditions_to_components
+from app.shared.data_classes import Condition
+from app.shared.data_classes import ConditionValue
+from app.shared.data_classes import SubCondition
+
+
+@pytest.mark.parametrize(
+    "input_condition,exp_result",
+    [
+        (
+            {
+                "displayName": "org type a",
```

Comment:
Can we add these as a constant from a variable ? to reduce the code lines

Reply:
Thanks - good shout, refactored!